python/cudf/cudf/_lib/table.pyx

```diff
@@ -0,0 +1,46 @@
+from cudf._lib.cudf cimport *
+
+
+cdef class Table:
+    """
+    Python wrapper around cudf::table
+    """
+    def __cinit__(self, columns=None):
+        cdef gdf_column* c_col
+        if columns is not None:
+            for col in columns:
+                c_col = column_view_from_column(col)
```

Comment:
Maybe the ability to set column names as well while doing `column_view_from_column`